src/components/Button/Button.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import './Button.scss';
+
+interface ButtonProps {
+  primary?: boolean;
+  size?: 'small' | 'medium' | 'large';
+  label: string;
+  classes?: string[];
+  backgroundColor?: string;
+  onClick?: () => void;
+}
+
+export const Button = ({
+  primary = false,
+  size = 'medium',
+  classes = [],
```

Comment:
Although this may be my experince only, but usually additional classes for the component are provided using the same "className" prop (as string, not array) as React itself utilizes. That makes the component API easier to learn and understand.